headers/Bullet.h

```diff
@@ -8,24 +8,25 @@ struct Environment;
 class Bullet final : public Pawn
 {
 public:
-	Bullet(const Point& pos, int width, int height, int color, float speed, Direction direction, int health, Environment* env);
+	Bullet(const Point& pos, int width, int height, int color, float speed, Direction direction, int health,
+		   Environment* env);
 
 	~Bullet() override;
 
-	void Move(Environment* env) override;
-	void Draw(const Environment* env) const override;
+	void Move() override;
+	void Draw() const override;
 	void KeyboardEvensHandlers(Environment& env, Uint32 eventType, SDL_Keycode key) override;
```

Comment:
need to fix arguments. (Must be: void KeyboardEvensHandlers(Uint32 eventType, SDL_Keycode key) override;

Reply:
now using internal field _env and for bullet method was marked as deleted